front/components/assistant_builder/SlackIntegration.tsx

```diff
@@ -52,7 +52,12 @@ export function SlackIntegration({
 
       setNewSelection((prevSelection) => {
         if (!prevSelection) {
-          return [];
+          return [
+            {
+              slackChannelId: internalId,
+              slackChannelName: title,
+            },
+          ];
```

Comment:
i remove this case and just did:

```
// Create a copy of the previous selection.
const updatedSelection = [...(prevSelection ?? [])];
```

Reply:
https://github.com/dust-tt/dust/pull/7074